fish-pepper/run-java-sh/fp-files/run-java.sh

```diff
@@ -601,7 +601,7 @@ run() {
   # Don't put ${args} in quotes, otherwise it would be interpreted as a single arg.
   # However it could be two args (see above). zsh doesn't like this btw, but zsh is not
   # supported anyway.
-  echo exec $(exec_args) java $(java_options) -cp "$(classpath)" ${args} "$@"
+  echo exec $(exec_args) java "$(java_options)" -cp "$(classpath)" ${args} "$@"
```

Comment:
You don't need the quotes here and it doesn't reflect the code which is executed.

Reply:
agreed and removed